api/src/data/seeds/015.data-ingestion-mappings-hss.ts

```diff
@@ -0,0 +1,152 @@
+    },
+    {
+      source_id: 1,
+      source_attribute: "Specific Event Type(s)",
+      source_value: null,
+      target_attribute: "description_moderated",
+      target_value: null,
+    },
+    {
+      source_id: 1,
+      source_attribute: "Severity Level (Reported)",
+      source_value: "Incident (minor)",
+      target_attribute: "incident_type_id",
+      target_value: "1",
+    },
+    {
+      source_id: 1,
+      source_attribute: "Severity Level (Reported)",
+      source_value: "Near Miss (serious)",
+      target_attribute: "incident_type_id",
+      target_value: "3",
+    },
+    {
+      source_id: 1,
+      source_attribute: "Severity Level (Reported)",
+      source_value: "Incident (serious)",
+      target_attribute: "incident_type_id",
+      target_value: "1",
+    },
+    {
+      source_id: 1,
+      source_attribute: "Severity Level (Reported)",
+      source_value: "Near Miss (minor)",
+      target_attribute: "incident_type_id",
+      target_value: "3",
+    },
+    {
+      source_id: 1,
+      source_attribute: "Severity Level (Reported)",
+      source_value: "Incident (minor)",
+      target_attribute: "urgency_code",
+      target_value: "Low",
+    },
+    {
+      source_id: 1,
+      source_attribute: "Severity Level (Reported)",
+      source_value: "Near Miss (serious)",
+      target_attribute: "urgency_code",
+      target_value: "Critical",
+    },
+    {
+      source_id: 1,
+      source_attribute: "Severity Level (Reported)",
+      source_value: "Near Miss (minor)",
+      target_attribute: "urgency_code",
+      target_value: "Low",
+    },
+    {
+      source_id: 1,
+      source_attribute: "Severity Level (Reported)",
+      source_value: "Incident (serious)",
+      target_attribute: "urgency_code",
+      target_value: "Critical",
+    },
+    {
+      source_id: 1,
+      source_attribute: "Division (responsible for the person)",
+      source_value: null,
+      target_attribute: "location_detail",
+      target_value: null,
+    },
+    {
+      source_id: 1,
+      source_attribute: "Branch",
+      source_value: null,
+      target_attribute: "location_detail",
+      target_value: null,
+    },
+    {
+      source_id: 1,
+      source_attribute: "Program",
+      source_value: null,
+      target_attribute: "location_detail",
+      target_value: null,
+    },
+    {
+      source_id: 1,
+      source_attribute: "Brief Factual Description",
+      source_value: null,
+      target_attribute: "description",
+      target_value: null,
+    },
+  ];
+
+  for (const row of mappings) {
+    const { source_id, source_attribute, source_value, target_attribute, target_value } = row;
+    const key = { source_id, source_attribute, target_attribute };
+    if (row.source_value == null) {
+      await knex("data_ingestion_mappings").where(key).whereNotNull("source_value").del();
+    } else {
+      await knex("data_ingestion_mappings").where(key).whereNull("source_value").del();
+    }
```

Comment:
Seeds should not be destructive. They should be idempontent and additive.
Why? Because then once these hit production, they will wipe any future data that looks like this, forever.

(Seeds generally run every time the app boots)